test/test_nnapi.py

```diff
@@ -450,6 +450,9 @@ def test_upsample_nearest2d(self):
     def test_linear(self):
         torch.manual_seed(29)
         self.check(torch.nn.Linear(16, 32), torch.randn(2, 16))
+        self.check(
+            torch.nn.Linear(16, 32), torch.randn(2, 16),
+            convert_args=[torch.zeros(0, 0)])
```

Comment:
I think this should be `zeros(0, 16)`.  We don't need to support flex dims on the second dimension, since that has to match the weight dimension, right?